CleanWeather/Scenes/CityDetails/CityDetailsDisplayable/CityDetailsListDisplayable.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  CityDetailsListDisplayable.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 21/11/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import Foundation
+
+struct CityDetailsListDisplayable {
+    
+    let id: String
+    let hour: [String]
+    let temp: [String]
+    let precip: [String]
```

Comment:
what is precip?

Reply:
I've checked it on google translate, I thought it's a shortcut, but it seems like a correct word. Let's leave precip :)